packages/frontend-shared/src/gql-components/useClipboard.ts

```diff
@@ -0,0 +1,37 @@
+import { Clipboard_CopyToClipboardDocument } from '../generated/graphql'
```

Comment:
Let's add a comment to this file, explaining why sending events to the server rather than just using browser APIs. Leave a nice little hint for the next person in here a couple years down the line. :)